src/nu/validator/checker/LanguageDetectingChecker.java

```diff
@@ -656,12 +656,13 @@ public void setHttpContentLanguageHeader(String httpContentLangHeader) {
      */
     @Override
     public void endDocument() throws SAXException {
-        if (!"0".equals(System.getProperty(
-                "nu.validator.checker.enableLangDetection"))
+        if (documentContent.length() > MIN_CHARS &&
```

Comment:
Isn't the check at https://github.com/validator/validator/blob/a177cd1b4c1fb324b4f2777beefe54142ea845b5/src/nu/validator/checker/LanguageDetectingChecker.java#L678-L680 sufficient?